### PR TITLE
Chore: pip install urllib3==1.24.2

### PR DESCRIPTION
**Pull request by yue9944882**
Fixes: #114

### CODE REVIEW

**Comment by irvifa** [comment]
@yue9944882 The workaround is deleting the dotnet related part the Dockerfile and basically your script fix the previous problem

File: openapi/openapi-generator/Dockerfile [hunk 1]
```diff
@@ -18,7 +18,7 @@ ARG GENERATION_XML_FILE
 ARG OPENAPI_GENERATOR_USER_ORG=OpenAPITools
 
 # Install preprocessing script requirements
-RUN apt-get update && apt-get -y install git python-pip && pip install urllib3
+RUN apt-get update && apt-get -y install git python-pip && pip install urllib3 && pip install requests[security]
```
Comment (roycaihw) [inline openapi/openapi-generator/Dockerfile]: ref https://github.com/kubernetes-client/gen/issues/114#issuecomment-493622236

**Comment by brendandburns** [comment]
@roycaihw do we think this is ready to merge? We don't have CI/CD here so someone needs to manually validate.

File: openapi/openapi-generator/Dockerfile (hunk 1 above)
Comment (yue9944882) [reply openapi/openapi-generator/Dockerfile]: updated the pip incantation to `urllib3==1.24.2`

**Comment by yue9944882** [comment]
i don't know how the magic works, but i believe it'll work perfectly

**Comment by irvifa** [comment]
It's working

**Pull request merged**